book/website/reproducible-research/renv/renv-containers.md

```diff
@@ -420,6 +430,120 @@ docker run -d --name dind-rootless --privileged docker:20.10-dind-rootless
 
 The following prequisites, which are part of the [`shadow-utils`](https://github.com/shadow-maint/shadow) package are required to run Docker rootless: `newuidmap` and `newgidmap`.
 
+(rr-renv-containers-podman)=
+## Podman
+
+[Podman](https://podman.io/) is an open-source container engine that can be seen as a compelling alternative to Docker.
+In fact, Podman provides a Docker-compatible command line interface.
+For most use cases a user familiar with Docker can simply alias the Docker command to Podman (`alias docker=podman`) and carry on as normal.
+Podman includes a [full set of tools](https://docs.podman.io/en/latest/Commands.html) to create, run and manage and share containers.
+
+Podman is free and open-source software released under the Apache License 2.0.
+This is in contrast to Docker which has some open-source components, such as the engine and command line interface, but also develops closed-source, [subscription-requiring](https://www.docker.com/blog/updating-product-subscriptions/) software including the Docker Desktop clients for MacOS and Windows.
+
+All [Open Container Initiative](https://opencontainers.org/) container images can be used with Podman including Docker images hosted on Docker Hub.
+It is likely existing projects using Docker can be migrated to Podman.
+
+Unlike Docker which uses a daemon running as root, Podman is daemonless.
+Unprivileged users can run containers using Podman.
+In most cases this will be configured automatically.
+This avoids a problem with the standard Docker configuration where users able to run containers have implicit access to the Docker daemon.
```

Comment:
No problem. On reflection I think it is unclear because it is hinting at something with properly explaining it. I have added the example above as a note in [5c3c907](https://github.com/alan-turing-institute/the-turing-way/pull/2370/commits/5c3c907c947879e4560bdc4b1927982be8bfab51) :+1: